config/application.rb

```diff
@@ -0,0 +1,27 @@
+require_relative "boot"
+
+require "rails/all"
+
+# Require the gems listed in Gemfile, including any gems
+# you've limited to :test, :development, or :production.
+Bundler.require(*Rails.groups)
+
+module OnlineClassroom
+  class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 7.2
```

Comment:
why we are using this?  `config.load_defaults 7.2`

Reply:
Bhai, I got to learn that `config.load_defaults 7.2` loads the default settings for Rails version 7.2, ensuring that the application is set up according to the best practices and configurations recommended for this specific version.
Thank you